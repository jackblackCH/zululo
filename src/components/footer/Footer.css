.c-footer {
  font-size: 10px;
  padding: 4px; }
  .c-footer__link {
    text-decoration: none; }
