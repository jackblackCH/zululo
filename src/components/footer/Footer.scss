.c-footer {
    font-size: 10px;
    padding: 4px;

    &__link {
      text-decoration: none;
    }
}
