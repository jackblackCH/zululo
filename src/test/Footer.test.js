import Footer from '../components/footer';
import renderer from 'react-test-renderer';
import React from 'react';


describe('Component: Footer', () => {

  it('should render the Footer', () => {
    const tree = renderer.create(
      <Footer></Footer>
    ).toJSON();
    expect(tree).toMatchSnapshot()
  })

  it('should have a className', () => {
    const tree = renderer.create(
      <Footer className='c-footer'></Footer>
    ).toJSON();
    expect(tree).toMatchSnapshot()
  })

  it('should have a link', () => {
    const tree = renderer.create(
      <Footer ><a className="c-footer__link" href="//github.com/jackblackCH/zululo"> Copyright  zululo 2017</a></Footer>
    ).toJSON();
    expect(tree).toMatchSnapshot()
  })

})
