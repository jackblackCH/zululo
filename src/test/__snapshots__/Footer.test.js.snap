exports[`Component: Footer should have a className 1`] = `
<footer
  className="c-footer">
  <a
    className="c-footer__link"
    href="//github.com/jackblackCH/zululo">
     Copyright  zululo 2017
  </a>
</footer>
`;

exports[`Component: Footer should have a link 1`] = `
<footer
  className="c-footer">
  <a
    className="c-footer__link"
    href="//github.com/jackblackCH/zululo">
     Copyright  zululo 2017
  </a>
</footer>
`;

exports[`Component: Footer should render the Footer 1`] = `
<footer
  className="c-footer">
  <a
    className="c-footer__link"
    href="//github.com/jackblackCH/zululo">
     Copyright  zululo 2017
  </a>
</footer>
`;
